CONTRIBUTING.txt

```diff
@@ -379,6 +379,27 @@ To rebuild a full clean documentation, run:
     make clean
     make html
 
+Gallery
+~~~~~~~
+
+If you are contributing an example to the
+`gallery <https://scikit-image.org/docs/dev/auto_examples/>`_ (or editing an
+existing one), write in
+`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_.
+Indeed, the gallery is a
+`Sphinx-Gallery <https://sphinx-gallery.github.io/stable/index.html>`_ one.
```

Comment:
"one" might be unnecessary

Reply:
Agreed! 😂

Reply:
The gallery is created using `Sphinx-Gallery <https://sphinx-gallery.github.io>`_. Refer to that page for complete instructions on syntax and usage.

Reply:
Thanks for this better wording! Included in bdfbbd2. I don't love my first sentence in the new [sub * (n + 1)]section ('The above-mentioned command includes the installation of Sphinx-Gallery.') but I thought repeating
```sh
pip install -r requirements/docs.txt
```
wouldn't be great either...